### PR TITLE
Add tcp load test for perf benchmark

### PR DESCRIPTION
**Pull request by daixiang0**
Fix https://github.com/istio/istio/issues/27647

Signed-off-by: Xiang Dai <long0dai@foxmail.com>

### CODE REVIEW

**Comment by daixiang0** [comment]
Since small machine, i just test run at local, no benchmark result here.

**Review by carolynhu — COMMENTED** [review]
could you update readme with this as well? thanks!

File: perf/benchmark/runner/runner.py [hunk 1]
```diff
@@ -93,6 +93,7 @@ def kubectl_exec(pod, remote_cmd, runfn=run_command, container=None):
 class Fortio:
     ports = {
         "http": {"direct_port": 8077, "port": 8080},
+        "tcp": {"direct_port": 8078, "port": 8078},
```
Comment (howardjohn) [inline perf/benchmark/runner/runner.py:96]: Should direct_port be the same as port?

Also a bit odd "grpc" and "direct_envoy" are the same port... but not related to this pr?

**Comment by daixiang0** [comment]
@carolynhu would update doc after #1315 merged.

File: perf/benchmark/runner/runner.py (hunk 1 above)
Comment (daixiang0) [reply perf/benchmark/runner/runner.py:96]: Not sure any side effect here, need @carolynhu help.

**Review by carolynhu — APPROVED** [review]

**Comment by ldemailly** [comment]
nice! thanks!

**Pull request merged**